pages/index.tsx

```diff
@@ -2,10 +2,9 @@ import Head from 'next/head';
 import Image from 'next/image';
 import EventCard from '../components/EventCard';
 import banner from '../public/banner.png';
-
```

Comment:
Is it expected that this space is removed?

Reply:
It's just a space, shouldn't matter